dist/21-526EZ-ALLCLAIMS-schema.json

```diff
@@ -908,14 +908,9 @@
   "properties": {
     "alternateNames": {
       "type": "array",
-      "minItems": 1,
```

Comment:
Ohh, I think I get it. This was here previously because _if_ `alternateNames` exists, it needs to have at least one item...which requires the `first` and `last` properties.